lib/matplotlib/tests/test_collections.py

```diff
@@ -325,6 +325,18 @@ def test__EventCollection__set_linestyle():
     splt.set_title('EventCollection: set_linestyle')
 
 
+@image_comparison(baseline_images=['EventCollection_plot__set_ls_dash'])
```

Comment:
add the "remove_text=True" kwarg and update the test image if you aren't going to test text. This helps deal with the tiny differences that occur with different versions of freetype.